src/sentry/search/base.py

```diff
@@ -14,36 +14,13 @@
 
 
 class SearchBackend(Service):
-    __all__ = frozenset([
-        'query',
-    ])
+    __read_methods__ = frozenset(['query'])
+    __write_methods__ = frozenset()
+    __all__ = __read_methods__ | __write_methods__
```

Comment:
See GH-8122 for precedent on the `__read_methods__` set. I added `__write_methods__` for consistency.